test/test_jit.py

```diff
@@ -10698,6 +10711,145 @@ def wait_script(x1, x2, x3):
         self.assertEqual(y2, foo2(x1, x2))
         self.assertEqual(y3, foo3(x1, x2, x3))
 
+
+loss_tests = [  # noqa: C408
```

Comment:
Couldn't we just use tests from common_nn instead of adding them here?

Reply:
Requires a fair amount of refactoring, can do it in a future diff